core/modules/menu/tests/menu.test

```diff
@@ -681,6 +681,7 @@ class MenuNodeTestCase extends BackdropWebTestCase {
     $edit = array(
       "title" => $node_title,
       "body[$language][0][value]" => $this->randomString(),
+      "menu[enabled]" => 0,
```

Comment:
can't tell if anything is wrong but since the test is using backdropPost it means it's simulating using the browser so you could look at HTML to see what options are available. If that's a node add form perhaps menu isn't even there? You'd then need to set that separately before this